docker_push.sh

```diff
@@ -1,5 +1,10 @@
 #!/bin/bash
 
-VERSION="1.2.5"
+set -e
+
+VERSION="$(git describe --tags --match 'v*')"  # Get version tag from git
```

Comment:
I was thinking maybe its better to have a more specific match pattern; at least making sure that 'v' is followed by a number. If we think there might be non-version tags, it's likely that a tag can be a word starting with 'v'. 

```suggestion
VERSION="$(git describe --tags --match 'v[:digit:]*')"  # Get version tag from git
```

Reply:
Sorry, didn't see this until now. The match flag takes a glob, not a regex.

```man
--match <pattern>
           Only consider tags matching the given glob(7) pattern, excluding the "refs/tags/" prefix.
```

Reply:
Sorry for the late comment. Yeah, if it was a regex we could actually match the complete version pattern. But, that's also a glob pattern to just match the first character of the tag after 'v' to be a number. According to 'man 7 glob', the `[[:digit:]]` pattern is also recognized, so I used it instead of `[0-9]` range.